apps/snitch_core/test/data/schema/order_test.exs

```diff
@@ -126,11 +126,13 @@ defmodule Snitch.Data.Schema.OrderTest do
       }
 
       params = %{
-        shipping_address: address_params
+        shipping_address: address_params,
+        billing_address: address_params
       }
 
       cs = Order.partial_update_changeset(persisted, params)
       refute cs.valid?
+
       assert %{shipping_address: %{state_id: ["does not exist"]}} = errors_on(cs)
```

Comment:
You also have to update this when you add the `billing_address` as `errors_on(cs)` is:
```elixir
%{
  shipping_address: %{state_id: ["does not exist"]},
  billing_address: %{state_id: ["does not exist"]},
}
```